src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/ColumnDefinitionCollectionConverter.cs

```diff
@@ -0,0 +1,112 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Windows.Controls;
+using System.Windows.Markup;
+using System.Globalization;
+using System.Text;
+using MS.Internal;
+
+namespace System.Windows.Controls
+{
+    internal sealed class ColumnDefinitionCollectionConverter : TypeConverter
```

Comment:
I wonder if it makes sense to have create base type converter for code shared between Rows and Columns

Reply:
They don't really have any properties in common, the code would have to be split anyway even if it was a single converter. And I would rather have two standalone converters than one relying on type descriptor to figure out what the converter should produce.

Reply:
I agree with @miloush here, there is very less commonality between the two properties and we would need to bifurcate the code to handle the properties.